.gitattributes

```diff
@@ -1,4 +1,4 @@
 *.tar.gz filter=lfs diff=lfs merge=lfs -text
 *.pdf filter=lfs diff=lfs merge=lfs -text
-*.txt filter=lfs diff=lfs merge=lfs -text
-*.gz filter=lfs diff=lfs merge=lfs -text
+data*.txt filter=lfs diff=lfs merge=lfs -text
```

Comment:
.seg files should also be tracked by lfs

Reply:
Removed that line for .gz file

Reply:
:+1: